Framework/LiveData/test/KafkaEventStreamDecoderTest.h

```diff
@@ -66,9 +66,8 @@ class KafkaEventStreamDecoderTest : public CxxTest::TestSuite {
     using namespace Mantid::LiveData;
 
     auto mockBroker = std::make_shared<MockKafkaBroker>();
-    Mock::VerifyAndClearExpectations(&mockBroker);
     EXPECT_CALL(*mockBroker, subscribe_(_, _))
-        .Times(Exactly(3))
+        .Times(Exactly(2))
```

Comment:
Looks good, but despite being restricted to 2 calls. It is set up to return 3 outputs.

Reply:
That was is mistake on my part. I was testing to make sure it would fail. Need to change this back.

Reply:
👍